test/test_helper.rb

```diff
@@ -4,6 +4,9 @@
 require 'mocha/setup'
 require 'hammer_cli'
 require 'hammer_cli_foreman/commands'
+require 'coveralls'
+
+Coveralls.wear! if RUBY_VERSION > "2.2"
```

Comment:
We run travis against multiple ruby versions but we only want coveralls to run once so here I am just having it run against 2.2.x ruby versions (or greater) which also matches katello-deploy.